_source/handbook/02_drive.md

```diff
@@ -311,4 +311,54 @@ Preload links into Turbo Drive's cache using `<a href="/" data-turbo-preload>Hom
 This will make page transitions feel lightning fast by providing a preview of a page even before the first visit. Use it to preload the most important pages in your application. Avoid over usage, as it will lead to loading content that is not needed.
 
 It also dovetails nicely with pages that leverage [Eager-Loading Frames](/reference/frames#eager-loaded-frame) or [Lazy-Loading Frames](/reference/frames#lazy-loaded-frame). As you can preload the structure of the page and show the user a meaningful loading state while the interesting content loads.
+
+## Ignored Paths
+
+Paths with a `.` in the last level of a path/URL will not be handled by Turbo unless they end in a file extension `.htm`, `.html`, `.xhtml`, or `.php`. Turbo will ignore forms and links that target these paths. For example, the following forms would be ignored:
```

Comment:
Perhaps here we could also explicitly say that if the URL ends in `/` it will be handled by Turbo. As it is now, if you don't notice the small detail of the final slash you wouldn't notice it. This would allow to put only 4 examples, 2 for those that are handled and 2 for those that aren't. Perhaps giving 4 examples for each case might be a bit excessive?